src/components/Spinner/index.tsx

```diff
@@ -12,9 +13,13 @@ type SpinnerProps = {
   centered?: boolean;
 };
 
-const Spinner: FC<SpinnerProps> = ({ size = 'lg', className, centered = false, ...rest }) => (
+const Spinner: FC<SpinnerProps> = ({ size = 'sm', className, centered = false, ...rest }) => (
```

Comment:
Is it possible to make a generalized component? The big spinner and small spinner images are different? I'd like to use the same image. @hetfly Please get help from @parrot9design if the images are different.

Reply:
thanks